src/v3/src/transformer/uischema/setLtrFields.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {
+  FieldElement,
+  TransformStepFn,
+} from '../../types';
+import { traverseLayout } from '../util';
+
+export const setLtrFields: TransformStepFn = (formbag) => {
+  traverseLayout({
+    layout: formbag.uischema,
+    predicate: (el) => {
+      if (el.type === 'Field') {
+        // fields with these names should always be LTR
+        const ltrOnlyFieldNames = ['credentials.passcode', 'identifier', 'credentials.newPassword', 'confirmPassword'];
+        const fieldElement = (el as FieldElement);
+        if (ltrOnlyFieldNames.includes(fieldElement.options.inputMeta.name)) {
+          return true;
+        }
```

Comment:
nit: a bit safer if we
```ts
return ltrOnlyFieldNames.includes(fieldElement.options.inputMeta.name)
```

Could also restructure the logic to initially check is `el.type !== 'Field'` and short-circuit `return false;` early